docs/design-docs/multi-DM.md

```diff
@@ -66,3 +70,57 @@ behaviors, namely:
      LockingTransactions.client - to ensure that the lock identifier
      is consistent across all replicas.  This change should be
      straightforward.
+     
+## AtLeastOnceRPC + DHT + ConsensusRSM
+
+### Desired behavior
+
+1. Client creates a new instance or obtains a reference to an existing MicroService.
+2. Client makes a call to an application method in Microservice and gets the result back.
+3. Client DM retries on failure for certain duration.
+4. Microservice replicas are sharded.
+5. Client expects the MicroService to be highly available,
+   resilient to server machine failures (provided that concurrent
+   failures are limited to a minority quorum).
+6. Client expects the MicroService to be high performance (all
```

Comment:
This is actually not implied by simply using ConsensusRSM.  Only if the necessary node selection is done.

Reply:
Removed.